docs/server-side/v2/introduction/webhook.mdx

```diff
@@ -243,37 +243,47 @@ This event will be sent when peer leaves the room
+| joined_at          | `timestamp (in UTC)` | Timestamp when user joined <br/><br/> Example: 2020-11-11T16:32:17Z                                                   |
+| session_started_at | `timestamp (in UTC)` | Timestamp when session started <br/><br/> Example: 2020-11-11T16:32:17Z                                               |
 
 #### Peer Leave Reason
 
-1. `client request`
-2. `peer kicked`
-3. `websocket closed`
-4. `ice connection state closed`
-
-Note:
-
-1. In case of peer is kicked / room is ended
-    - Client itself initiates a peer leave on receiving the signal. In these cases, reason will be `client request`
-    - Also, server starts a timer for the same. If peer leave is not received from client before the timer ends, the server then kicks the peer itself. In such cases, reason will be `peer kicked`
-2. All other reasons except `client request` and `peer kicked` denote that the peer has left unexpectedly.
+1. `client request`: if the peer leaves the room (leave request is sent from the app/client-side).
+    - _Example scenarios:_
+        - If the peer clicks the `Leave/End` button.
+        - If the peer refreshes or closes the tab while using one of 100ms demo links (`https://<YOUR_SUB_DOMAIN>.app.100ms.live/preview/nnd-eke-tpp`)
+2. `peer kicked`: if the peer is removed by someone else from an active room using the [Client SDK](/javascript/v2/features/remove-peer) or the [Server API](/server-side/v2/active-rooms/remove-peers).
+    - _Example scenarios:_
+        - If the peer is removed by a role which has `Can remove participant from the room` [permissions](./../policy/create-template-via-dashboard#permissions) enabled in the template.
+        - If the peer is removed using the [remove peer API](./../active-rooms/remove-peers) from the application server. <br />  
+          **Note**: If you remove a peer from an [active room](./../active-rooms/overview) using the
+          [client SDK](/javascript/v2/features/remove-peer) or the [server API](/server-side/v2/active-rooms/remove-peers),
+          you can pass the reason as a message and the `peer.leave.success` event will contain the
+          same in the `message` field.
+3. `websocket closed`: network issues
+    - _Example scenario:_
+        - If there's an abrupt network disconnection from the client side (peer) without any further attempts to reconnect, 100ms server will close the WebSocket connection.
+4. `ice connection state closed`: network issues
+    - _Example scenario:_
+        - When a peer is disconnected due to network issues, the SDK will retry for 60 seconds and disconnect if there are no attempts to reconnect.
+5. `server removed peer`: network issues
+    - _Example scenario:_
+        - When a peer is removed by the server as there were no attempts to reconnect for 60 seconds and if the client-side app still sends retries after 60 seconds. This happens rarely.
```

Comment:
This will happen only if there are any server side anomalies like a server failure mid way into call will remove peers with this error. Occurrence of this is rare.